src/components/SecondaryPanes/Breakpoints/BreakpointsContextMenu.js

```diff
@@ -258,4 +258,7 @@ export default function showContextMenu(props: Props) {
   ];
 
   showMenu(contextMenuEvent, buildMenu(items));
+
+  // This is needed for flow to pass BreakpointsContextMenu.spec.js
```

Comment:
i don't think we need this comment... it'll likely go stale and if someone removes it flow will tell them they need it

Reply:
OK, I'll remove this.  I added this comment because the error message flow gives is pretty bizarre.

Reply:
i see... yeah, that can be difficult